core/server/common/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -145,6 +149,8 @@ public void start(Boolean isPrimary) throws IOException {
         }
       }
       mAsyncJournalWriter = new AsyncJournalWriter(mJournalWriter);
+      mAsyncAuditLogWriter = new AsyncUserAccessAuditLogWriter();
+      mAsyncAuditLogWriter.start();
```

Comment:
Since Alluxio writes audit log to local storage, therefore currently we start the audit logging thread only on primary master. In the future, we may use similar techniques to distributed storage or RAFT as a lightweight, reliable storage for audit information. This depends on customer needs.